eahub/templates/eahub/edit_profile.html

```diff
@@ -44,6 +44,10 @@
 
 {{ form.summary|as_crispy_field }}
 
+{{ form.linkedin_url|as_crispy_field }}
```

Comment:
If you implement URL validation (see comment above), make sure that appropriate error messages appear here when a user enters an invalid input

Reply:
This works automatically when I use `URLField`:

![image](https://user-images.githubusercontent.com/1735266/63232813-525a5e80-c1e0-11e9-8aa2-98ea6d58abd6.png)